python/tank/platform/qt/login.ui

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>LoginDialog</class>
+ <widget class="QDialog" name="LoginDialog">
```

Comment:
In the convention that is used in most places, the dialog name is in sync with the file name:

- UI file: `login.ui`
- Dialog name: `Dialog`
- Generated py file: `ui_login.py`
- Object inside `ui_login.py` file: `Ui_Login`
- Main dialog code file: `dialog.py` which then imports `Ui_Login`

Actually, maybe all we need to do is to rename `login.ui` and `ui_login.py` to make it all line up?